### DISCUSSION TITLE
Cascading Menu Button

### DISCUSSION THREAD

**Discussion by Jimmeywho** [TomSchimansky/CustomTkinter#1024] [Q&A]
I would like to create a cascading menu button, like the top file menu you get in application's. I've looked at bringing in tkinter's standard menu function but it does not like images. Ideally I'd like to use a png icon rather than text.
I've looked at the drop-down and combo box options and whilst they provide the cascading element, they're not quite right due to the variable display.
Has anyone encountered and resolved this challenge?
If not, any advice on splicing a button function with a drop-down element?
Thank you in advance!!

**Comment by LucianoSaldivia** [comment]
I've been trying this for a couple of months already, and while i made some progress, some details i just can't get to manage properly, like the closing of said menu (and children menues) when the user clicks outside the area for example. Every time i get into one of this issues i start searching for any python GUI with this feature, and except for old awful-looking plain tkinter, noone else seems to have implemented it yet...
Have you implemented something or found any solution for this requirement?
